### ISSUE TITLE
xplorer (95157.805) spins around in game and the pro mode icons do not show up in instrument picker

### ISSUE THREAD

**Issue by stringbender1** [InvoxiPlayGames/FestivalInstrumentMapper#45]
even with the xbox 360 controller hidden with the hid program it does still not work.

**Comment by stringbender1** [comment]
also, in my log i get this
LogGameInput: [FGameInputWindowsModule::CreateInputDevice] Failed to get the IGameInput object from the base module! You will not be able to process input.

and this LogGameInput: Warning: Failed to initialize GameInput: 0x80004001

i looked this up online and it says i need to uninstall gameinput but i cannot find that anywhere

**Comment by Believeinsteve92** [comment]
does your game spin around if you slowly push down on the whammy bar, and then start turning the other direction? This is what mine does on rb guitars.

**Comment by GalaxisBeast** [comment]
if you are spinning around, you need HIDHide to hide controllers from fortnite. if it isnt hiding it will see your guitar as regular controller and whammy makes you turn

**Comment by Believeinsteve92** [comment]
The issue afterwards at least for RB guitars is HIDHide will stop the whammy spinning camera but no inputs work in game. It sounds like HIDHide is supposed to see two controllers essentially and disable the "xbox 360 controller" one and not the guitar controller, but its only seeing the xbox 360 controller.

**Comment by stringbender1** [comment]
Yeah but my true problem is the gameinput api failing in fortnite

**Comment by InvoxiPlayGames** [comment]
If Fortnite's GameInput isn't able to be initialised then Xbox One guitars won't show up. Are you running the latest Windows Update and have you ever run any "debloating" scripts? Those can sometimes remove the Xbox input services.

**Comment by stringbender1** [comment]
I have never ran a debloating script and yes I am on the latest version of windows 10

**Comment by stringbender1** [comment]
damn still not working and yes there is gameinput in services and yes it is on automatically
